### PR TITLE
Branch overview Tab 

### PR DESCRIPTION
**Pull request by sakshi-nagpal**
Screenshot 2020-04-02 at 9 20 22 PM

### CODE REVIEW

File: ui/app.js [hunk 1]
```diff
@@ -60,7 +66,8 @@ class App extends React.Component {
 
   setBranch(branchName) {
     this.setState({ currentBranchName: branchName });
-    sendMessage({messageId: 'get-pull-request', data: {branchName: this.state.currentBranchName}});
+    //todo handled by the component
+    // sendMessage({messageId: 'get-pull-request', data: {branchName: this.state.currentBranchName}});
```
Comment (nishants) [inline ui/app.js:70]: Can we remove this comment?

File: ui/app.js [hunk 2]
```diff
@@ -49,8 +55,8 @@ class App extends React.Component {
     });
     // TODO : Just to test, remove this
     sendMessage({messageId: 'get-current-branch-info'});
-    sendMessage({messageId: 'get-pull-request', data: {branchName: "xyz-branch"}});
-    sendMessage({messageId: 'get-cypress-builds', data: {branchName: "xyz-branch"}});
+    // todo remove sendMessage({messageId: 'get-pull-request', data: {branchName: this.state.currentBranchName}});
```
Comment (nishants) [inline ui/app.js:58]: Can we remove this comment?

**Comment by nishants** [comment]
Great work Sakshi !!
Can you please merge with master and test on local before merging ?

File: ui/app.js (hunk 1 above)
Comment (sakshi-nagpal) [reply ui/app.js]: done

File: ui/app.js (hunk 2 above)
Comment (sakshi-nagpal) [reply ui/app.js]: done

**Pull request merged**